README.md

```diff
@@ -61,16 +31,18 @@ For more information on the importance of a professional README for your reposit
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 [BookStore] <a name="about-project"></a>
 
-**[your_project__name]** is a...
+**[BookStore]** is a project that uses a Single Page Application format to display items on a page. A user can switch to two pages wherein one displays the titles and authors of books, and also a form to add a new book.
 
 ## 🛠 Built With <a name="built-with"></a>
 
+-HTML
+-CSS
+-JavaScript
+-ReactJS
+-Redux
+
 ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [ ] Please delete the template placeholder text: ` > Describe the tech stack and include only the relevant sections that apply to your project.` in your readme, this will make your readme more professional 👍🏿